Cargo.toml

```diff
@@ -8,6 +8,6 @@ members = [
 # ark-serialize = { git="https://github.com/simonmasson/algebra", rev="e2ea75c" }
 # ark-ff = { git="https://github.com/simonmasson/algebra", rev="e2ea75c" }
 # ark-ec = { git="https://github.com/simonmasson/algebra", rev="e2ea75c" }
-halo2_gadgets = { git="https://github.com/heliaxdev/halo2", branch="taiga"}
-halo2_proofs = { git="https://github.com/heliaxdev/halo2", branch="taiga"}
-pasta_curves = { git="https://github.com/heliaxdev/pasta_curves", branch="taiga"}
+halo2_gadgets = { git = "https://github.com/heliaxdev/halo2", branch = "taiga" }
+halo2_proofs = { git = "https://github.com/heliaxdev/halo2", branch = "taiga" }
+pasta_curves = { git = "https://github.com/heliaxdev/pasta_curves", branch = "taiga" }
```

Comment:
Could we remove these patches as we've specified them in dependencies?

Reply:
No, unfortunately we can't. Unlike what we indicate in out dependencies, these patches also affect the whole dependency tree.